### DISCUSSION TITLE
New component to show snippets

### DISCUSSION THREAD

**Discussion by Alienah** [vmware-clarity/core#78] [Ideas]
Hello! 

First of all, congratulations for the work you do.

In our team we want to use Clarity components to create a theme that follows your styles to generate technical documentation. 

Since the content is technical, it would be very useful some component or proposal from Clarity to show snippets for the code. Are you working or do you plan to work on something to show snippets?

Thanks in advance

**Comment by ashleyryan** [comment]
we have the `cds-text=code` attribute, which you can put on a code tag. It's documented here:

https://clarity.design/storybook/core/?path=/docs/stories-typography--code

Here's an example:

https://stackblitz.com/edit/clarity-cdn-bdwase?file=index.html

We don't have any plans at this time to develop a more robust code sample component with syntax highlighting, if that's what you were asking for

**Reply by Alienah** [reply]
Thank you very much for your response. Indeed, the question was about having that code sample component with syntax highlighting.
